airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/context.py

```diff
@@ -286,3 +295,12 @@ async def __aexit__(
 
     def create_slack_message(self) -> str:
         raise NotImplementedError
+
+    def _get_step_id_to_skip_according_to_metadata(self) -> List[CONNECTOR_TEST_STEP_ID]:
+        enabled_test_suites = [option["suite"] for option in self.metadata.get("connectorTestSuitesOptions", [])]
+        return [step_id for test_suite_name, step_id in TEST_SUITE_NAME_TO_STEP_ID.items() if test_suite_name not in enabled_test_suites]
+
+    def _get_updated_run_step_options(self, run_step_options: RunStepOptions) -> RunStepOptions:
```

Comment:
❗ The name of the function explains the how but not the what. I'd propose we change it to


```suggestion
def _skip_metadata_disabled_test_suites(self, run_step_options: RunStepOptions) -> RunStepOptions:
```